projects/RabbitMQ.Client/client/impl/InlineConsumerDispatcher.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Buffers;
+using System.Threading.Tasks;
+using RabbitMQ.Client.Impl;
+
+namespace RabbitMQ.Client.client.impl
+{
+    internal sealed class InlineConsumerDispatcher : IConsumerDispatcher
```

Comment:
This class will need to call Model.OnCallbackException to conform to other dispatcher.

Reply:
oki I will have a look at the other dispatchers to implement that

Reply:
and add a test that verifies this behavior because right now all tests are green except public api change